### ISSUE TITLE
Blank space after tablesorter renders

### ISSUE THREAD

**Issue by Alchaves** [Mottie/tablesorter#1465]
Hi,

I have a div with a spinner gif that it is showing, while the javascript/css is being computed and a div with the content of the page is hidden. 
And when everything is ready, a piece of javascript hides the div with the spinner and shows the div with the content.

The problem is that because the tablesorter initializes in a hidden div, it doesn't know how much space it have/is going to ocupy, therefor it sets the height for all the rows even if after the pager shortened the final height of the tablesorter, leaving a lot of blank space after the body of the document.

The weirdest thing is that I can still adjust width of the columns on that blank space...

My question is, is there any setting or fix for this bug?

PS: taking the div with the spinner gif out, is not possible...

Thanks in advance.
Hope I've been clear with the problem in question.

**Comment by Mottie** [comment]
Hi @Alchaves!

Please share which widgets and/or addons you're using because I can't think of one that sets the height of the rows. The only thing I can think of is that the original tablesorter (v2.0.5) included the pager addon which calculated the table height and then positioned the controls at the bottom. This fork doesn't do that.

If you are using this fork's pager, then please [modify this demo](https://jsfiddle.net/Mottie/wty134u7/) to help me troubleshoot the problem.

**Comment by Alchaves** [comment]
Hi @Mottie!

The tablesorter version is the v2.28.5 with the following widgets/addons:
lazyload;
print;
pager;

It isn't the original version, but it is pretty old...
I do believe it as something to do with the pager addon, but I can't have any certain of that...

**Comment by Mottie** [comment]
Please provide a demo of the problem, and/or share some code, to help me troubleshoot the problem.

**Comment by Alchaves** [comment]
Hi @Mottie!

Sorry for the late response (I was pulled way to another part of the current project), but I finally figured it out why I wasn't abble to reproduce the problem in your demo.

It seems that in your latest version, the problem doesn't happen.

I tested with your latest version and everything is working just perfectly.

We will try to update our version soon, to the latest version.

Thanks again for your help and keep up the good work.
